### ISSUE TITLE
New OEIS sequence causes AxiosError: Network Error

### ISSUE THREAD

**Issue by katestange** [numberscope/frontscope#206]
To reproduce:  Pick a totally random A-number the database has not seen before.  Add it as an OEIS sequence.  Wait 1-2 seconds.  Then up pops a "AxiosError:  Network Error".  Reload the page and try adding that OEIS sequence again.  No problems now.

**Comment by gwhitney** [comment]
This should be revisited once https://github.com/numberscope/backscope/pull/127 lands and is installed on the production server.

**Comment by gwhitney** [comment]
Duplicate of #407, closing.
